templates/eventwizard.html

```diff
@@ -202,7 +202,7 @@ <h3>Alliance Selection</h3>
 {% block inline_javascript %}
 <script src="//cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.2/rollups/md5.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-csv/0.71/jquery.csv-0.71.min.js"></script>
-<script src="//cdnjs.cloudflare.com/ajax/libs/xlsx/0.8.0/xlsx.core.min.js"></script>
+<script src="//unpkg.com/xlsx/dist/xlsx.full.min.js"></script>
```

Comment:
why does this need to be changed?

Couldn't `https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.14.0/xlsx.full.min.js` work just as well?